arcaflow_plugin_stressng/stressng_plugin.py

```diff
@@ -133,14 +113,14 @@ def stressng_run(
         os.remove(stressng_jobfile[1])
 
     return "success", WorkloadResults(
-        params,
-        system_un,
-        vminfo_un,
-        mmapinfo_un,
-        cpuinfo_un,
-        matrixinfo_un,
-        mqinfo_un,
-        hddinfo_un,
+        test_config=params,
+        systeminfo=system_un,
+        vminfo=results.get(Stressors.VM),
+        mmapinfo=results.get(Stressors.MMAP),
+        cpuinfo=results.get(Stressors.CPU),
+        matrixinfo=results.get(Stressors.MATRIX),
+        mqinfo=results.get(Stressors.MQ),
+        hddinfo=results.get(Stressors.HDD),
     )
```

Comment:
@dustinblack, shouldn't this parameter list include entries for `iomix` and `sock`??